tests/core/engine_adapter/test_trino.py

```diff
@@ -148,26 +151,36 @@ def test_partitioned_by_iceberg_transforms(
         return_value="datalake_iceberg",
     )
 
-    columns_to_types = {
-        "cola": exp.DataType.build("INT"),
-        "colb": exp.DataType.build("TEXT"),
-    }
+    expressions = d.parse(
+        f"""
+        MODEL (
+            name test_table,
+            partitioned_by (day(cola), truncate(colb, 8), colc),
```

Comment:
Is this the correct syntax for specifying multiple partition columns in a SQL model? I couldn't actually find it documented anywhere. It seems to work but im not sure if I should be using `[ ]` array syntax instead